connectivity/FEATURE_BLE/mbed_lib.json

```diff
@@ -2,6 +2,16 @@
     "name": "ble",
     "config": {
         "present": 1,
+        "ble-api-implementation": {
+            "help": "What BLE user API implementation is used. Available options: 1 (Mbed). You may add more. See headers in include/ble/.",
+            "value": 1,
+            "macro_name": "BLE_API_IMPLEMENTATION"
+        },
+        "ble-pal-api-implementation": {
+            "help": "What Pal API implementation is used. Available options: 1 (Cordio). You may add more. See headers in include/ble/internal/Pal*.",
+            "value": 1,
+            "macro_name": "BLE_PAL_API_IMPLEMENTATION"
+        },
```

Comment:
See my comment above. If impl header names are standardised, we can always assume they exist.
The TARGET_CORDIO magic name already does the switch to use Cordio (i.e. default) impl or not.

Reply:
yep, let me remove that